### DISCUSSION TITLE
Spam Checks : Timed out when scanning for spam

### DISCUSSION THREAD

**Discussion by amisalabir** [postalserver/postal#1634] [Help with using Postal]
Spamd is configured and active but Postal I gets this message "Timed out when scanning for spam". Did anyone face such issue. Please help if any of you solve this before.
![Screen Shot 2021-10-08 at 7 40 13 PM](https://user-images.githubusercontent.com/1482502/136568677-d6167914-5ffe-49c3-b57a-ffb11add0e17.png)
s.

**Comment by willpower232** [comment]
You should check your configuration and restart Postal as necessary

**Reply by amisalabir** [reply]
@willpower232 thank you for your answer. Please can you explain what should I check in the configuration?

**Reply by amisalabir** [reply]
Please can you watch at a glance my postal config as follows:
=============================================
general:
  # This can be changed to allow messages to be sent from multiple IP addresses
  use_ip_pools: true

web:
  # The host that the management interface will be available on
  host: postal.mydomain.com
  # The protocol that requests to the management interface should happen on
  protocol: https

web_server:
  # Specify configuration for the Postal web server
  bind_address: 127.0.0.1
  port: 5000

smtp_server:
  # Specify configuration to the Postal SMTP server
  port: 25

logging:
  # Specify options for the logging
  stdout: true

main_db:
  # Specify the connection details for your MySQL database
  host: 127.0.0.1
  username: root
  password: postal
  database: postal

message_db:
  # Specify the connection details for your MySQL server that will be house the
  # message databases for mail servers.
  host: 127.0.0.1
  username: root
  password: postal
  prefix: postal

rabbitmq:
  # Specify connection details for your RabbitMQ server
  host: 127.0.0.1
  username: postal
  password: postal
  vhost: postal

dns:
  # Specify the DNS records that you have configured. Refer to the documentation at
  # https://github.com/atech/postal/wiki/Domains-&-DNS-Configuration for further
  # information about these.
  mx_records:
    - mx.postal.mydomain.com
  smtp_server_hostname: postal.mydomain.com
  spf_include: spf.postal.mydomain.com
  return_path: rp.postal.mydomain.com
  route_domain: routes.postal.mydomain.com
  track_domain: track.postal.mydomain.com

smtp:
  # Specify an SMTP server that can be used to send messages from the Postal management
  # system to users. You can configure this to use a Postal mail server once the
  # your installation has been set up.
  host: 127.0.0.1
  port: 2525
  username: # Complete when Postal is running and you can
  password: # generate the credentials within the interface.
  from_name: Postal
  from_address: postal.mydomain.com

  spamd:
  enabled: true
  host: 127.0.0.1
  port: 783

rails:
  # This is generated automatically by the config initialization. It should be a random
  # string unique to your installation.
  secret_key: xxxxxxxxxxxxxxxxxxxx
fast_server:
  enabled: true
  bind_address: xxx.xxx.xxx.xxx

**Reply by willpower232** [reply]
>spamd:
>  enabled: true
>  host: 127.0.0.1
>  port: 783

You can use `sudo netstat -lnp` to verify that spamd is running on the same server and on that port.

**Reply by amisalabir** [reply]
![Screen Shot 2021-10-11 at 11 32 59 AM](https://user-images.githubusercontent.com/1482502/136738163-74ff0bb2-2afa-475d-b928-398136b2d44c.png)
Please can you check this? Thanks @willpower232

**Reply by amisalabir** [reply]
![image](https://user-images.githubusercontent.com/1482502/136765249-d50b37f3-5fa1-4d27-98c2-34e1ffd991e6.png)

Now I see this message.

**Reply by amisalabir** [reply]
Please can you tell me what does it mean

**Reply by willpower232** [reply]
I can only guess that spamd is not working right but I don't know enough about it, it looks like you're doing everything correctly.

**Reply by amisalabir** [reply]
Thanks
